base/gfspvgmgr/sp_freeze_pool.go

```diff
@@ -9,8 +9,8 @@ import (
 )
 
 const (
-	DefaultFreezingPeriodForSP = 10 * time.Minute
-	ReleaseSPJobInterval       = 1 * time.Minute
+	DefaultFreezingPeriodForSP = 2 * time.Second
```

Comment:
why does these interval or period become so frequently?

Reply:
this is reverted, just for local testing.